ruby/examples/app/controllers/user_registration_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserRegistrationController < ApplicationController
+  def new
+    @user = UserRegistration.new
+  end
+
+  def create
+    request = RegisterUser::Request.new(*user_params)
+    response = RegisterUser.new.register(request)
```

Comment:
I'd prefer to avoid boilerplate and dependency injection. It's especially not necessary as our dependencies for the use case are hard coded in it's constructor so the controller won't be a magnet for dependencies.